src/docs/product/dashboards/widget-library/index.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Widget Library
+sidebar_order: 1
+redirect_from:
+ - /product/dashboards/customize-dashboards/
+description: "Learn how to edit or create additional dashboards to suit your organization's needs."
+---
+
+<Note>
+
+This feature is available only if your organization is on either a Business or Trial plan.
+
+</Note>
+
+The Widget Library contains a collection of prebuilt widgets you can add to your [Custom Dashboards](/product/dashboards/customize-dashboards/). To access the Widget Library, click on "Add Widget(s)" and switch to the "Widget Library" tab on the modal that appears.
+
+The following widgets can be found in the library:
+
+- **Distribution Duration**: A multi-y axis chart visualizing the distribution of transaction duration across various percentiles (p50, p75 and p95)
+- **High Throughput Transactions**: The top 5 transactions with the largest number of events
+- **LCP by Country**: World map showing the p75 of page load times for each country
+- **Miserable Users**: The total number of unique users who have experienced slow transactions (transaction duration greater than 1200ms)
+- **Slow vs Fast Transactions**: Bar chart comparing the percentage of transactions that are over 300ms (slow) and under 300ms (fast)
+- **Latest Unresolved Issues**: A table of unresolved issues ordered by the most recently seen issues
+- **Top Unhandled Error Types**: The top 5 most frequently encountered unhandled errors
+- **Users Affected by Errors**: A comparison of the total number of errors and the number of unique users affected by the errors
+
+You can change the title, queries, fields, visualization types and sort order of these prebuilt widgets to suit your use case by clicking on the context menu on the widget and selecting "Edit Widget".
+
+## Example Use Cases
+
+### Performance across release versions
+
+To monitor how your app is performing on a certain release, you can modify the **Distribution Duration** widget by adding the release version to the search condition.
```

Comment:
Using bold on these widget names makes sense in the context of the bullet list to make them pop, but here, we're just treating them as UI elements so they're no-bold w quotation marks.

```suggestion
To monitor how your app is performing on a certain release, you can modify the "Distribution Duration" widget by adding the release version to the search condition.
```